### ISSUE TITLE
Hover text in commands doesn't work when supplying suggestions

### ISSUE THREAD

**Issue by JorelAli** [JorelAli/1.13-Command-API#35]
**My code**
This code reproduces the behavior in 1.8 (unreleased):
```java
return RequiredArgumentBuilder.argument(argumentName, type).suggests(provider);
```

This code fixes the behaviour:
```java
return RequiredArgumentBuilder.argument(argumentName, type);
```

**Expected behavior**
The hover text is shown when `suggests()` is used

**Screenshots**
This should be showing, even when suggests() is used (perhaps unless **permission is granted?** May conflict with #26)
![image](https://user-images.githubusercontent.com/4613171/50321162-67eb7d00-04c7-11e9-942d-af88ab3152a9.png)

**Comment by JorelAli** [comment]
You're creating a new `SuggestionProvider`. When you do this using solely a String[], you ignore the tooltip:

`builder.suggest(str);` isn't enough, you need to include the Message tooltip:
`builder.suggest(str, Message tooltip);`

**Comment by JorelAli** [comment]
Fixed with 1.8 due to the fix for #26
